Library/Homebrew/cmd/update-report.rb

```diff
@@ -218,16 +218,18 @@ def report
         new_tap = tap.tap_migrations[name]
         @report[status.to_sym] << full_name unless new_tap
       when "M"
-        begin
-          formula = Formulary.factory(tap.path/src)
-          new_version = formula.pkg_version
-          old_version = FormulaVersions.new(formula).formula_at_revision(@initial_revision, &:pkg_version)
-          next if new_version == old_version
-        rescue FormulaUnavailableError
-          # Don't care if the formula isn't available right now.
-          nil
-        rescue Exception => e # rubocop:disable Lint/RescueException
-          onoe "#{e.message}\n#{e.backtrace.join "\n"}" if Homebrew::EnvConfig.developer?
+        unless Homebrew.args.preinstall?
```

Comment:
```suggestion
        next if Homebrew.args.preinstall?
```

Reply:
This would skip adding modified formulae to the reporter, meaning that `update-report` commands that check the reporter for changed formulae wouldn't have the right data. For example `DescriptionCacheStore` would be out of date.